main.cpp

```diff
@@ -52,85 +52,86 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: Car
+//  action 1: turns left
+car.turnLeft();
+//  action 2: accelerates
+car.accelerate();
+//  action 3: displays low on fuel-warning 
+car.displayLowOnFuelWarninig(); 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Cat
+//  action 1: eats
+cat.eat();
+//  action 2: sleeps
+cat.sleep();
+//  action 3: catches a mouse
+cat.catchMouse();
  
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: Plane
+//  action 1:  starts engine
+plane.startEngine();
+//  action 2: changes the height
+plane.changeHeight();
+//  action 3: changes the speed
+plane.changeSpeed(); 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: student
+//  action 1: reads instruction
+student.readInstruction();
+//  action 2: asks for help
+student.askForHelp();
+//  action 3: sends a pull request
+student.sendPullRequest(); 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: printer
+//  action 1: print document
+printer.printDocument();
+//  action 2: scans document
+printer.scanDocument();
+//  action 3: creates an error
+printer.createError(); 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: teacher
+//  action 1: teaches
+teacher.teach();
+//  action 2: answers a question
+teacher.answerQuestion();
+//  action 3: writes an assignement
+teacher.writeAssignement();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: water
+//  action 1: boils
+water.boil();
+//  action 2: freezes
+water.freeze();
+//  action 3: makes things wet
+water.makeWet(); 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: child
+//  action 1: plays
+child.play();
+//  action 2: laughs
+child.laugh();
+//  action 3: cries
+child.cry(); 
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: calculator
+//  action 1: multiply
+calculator.multiply();
+//  action 2: subtract
+calculator.subtract();
+//  action 3: add
+calculator.add();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.